pkg/cmd/publish.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/appvia/artefactor/pkg/docker"
+	"github.com/spf13/cobra"
+)
+
+const (
+	FlagDockerRegistry string = "docker-registry"
+	// PublishCommand is the sub command syntax
+	PublishCommand string = "publish"
+)
+
+// publishCmd represents the version command
+var publishCmd = &cobra.Command{
+	Use:   PublishCommand,
+	Short: "publishes artefact(s)",
+	Long:  "will publish artefact(s) to correct registries / locations.",
+	RunE: func(c *cobra.Command, args []string) error {
+		return publish(c)
+	},
+}
+
+func init() {
+	addFlagWithEnvDefault(
+		publishCmd,
+		FlagArchiveDir,
+		DefaultArchiveDir,
+		"a directory where artefacts exist to publish from")
+
+	addFlagWithEnvDefault(
+		publishCmd,
+		FlagDockerRegistry,
+		"",
+		"where to publish images e.g. myrestirty.com/path")
```

Comment:
was this meant to say myregistry? :)

Reply:
Yep - updated to be correct and more descriptive.